MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -38,16 +40,15 @@
 
 @interface MSALPublicClientApplication : NSObject
 
+@property (readonly, nonnull) MSALPublicClientApplicationConfig *configuration;
```

Comment:
Add a comment about what it is.

Reply:
Sure

Reply:
Parameter to be used to configure MSALPublicClientApplication.
    It contains all values to be used in the instance and is a superset of all properties
    known to this class.